### ISSUE TITLE
[IMPROVE] Make the quote clickable and redirect to the message that was quoted

### ISSUE THREAD

**Issue by ritwizsinha** [RocketChat/Rocket.Chat#16298]
### Description:

Currently the quote isn't clickable but the date and time right next to it redirects to the message, It would be great if clicking anywhere on the quote redirects to the message

### Steps to reproduce:

1. Quote a message
2. Try to click on the markdown box that appears

### Expected behavior:

The quoted message should be clickable

### Actual behavior:

![out](https://user-images.githubusercontent.com/43509699/72776522-81b24300-3c38-11ea-96c7-986f1d572703.gif)

**Comment by ritwizsinha** [comment]
The same can be done for the pinned messages

**Comment by manosriram** [comment]
@ritwizsinha Can I work on this? Seems interesting!

**Comment by ritwizsinha** [comment]
Thanks @manosriram for showing interest but I am already working on it and will submit a pr soon

**Comment by manosriram** [comment]
@ritwizsinha Yeah ok.

**Comment by gabriellsh** [comment]
@manosriram Of course you can work on this! Thanks for the help man.

**Comment by ritwizsinha** [comment]
Yeah @manosriram sorry for this, I thought issues were to be claimed before working

> Thanks @manosriram for showing interest but I am already working on it and will submit a pr soon . Please check out other open issues

**Comment by Sing-Li** [comment]
@ritwizsinha   you should not claim an issue.  We welcome all contributions.

**Comment by ritwizsinha** [comment]
Sorry @Sing-Li  I didn't know that then

> @ritwizsinha you should not claim an issue. We welcome all contributions.

**Comment by manosriram** [comment]
@ritwizsinha It's Ok, nothing to worry! @gabriellsh @Sing-Li Thank you Sir.

**Comment by engelgabriel** [comment]
Let's add an ICON with a button beside the date, that is already a link:

![image](https://user-images.githubusercontent.com/1000217/82591315-8b5f9f00-9b75-11ea-9476-452c88c45197.png)

**Comment by serviceman** [comment]
This is nice, and I would love it, but Quotes aren't working at all in the version  3.4.2, shouldn't that be fixed first?

**Comment by Darshilp326** [comment]
@gabriellsh Do we need to add button as mentioned in https://github.com/RocketChat/Rocket.Chat/issues/16298#issuecomment-632261740 ?

**Comment by dougfabris** [comment]
Hi there, thanks for the contribution! 🚀 💯 

Sorry, this error does not happen on the latest and maintained releases. It was fixed here #41091

[Rocket.Chat](https://rocket.chat/)  provides support for, at least, 3 months for each release and 6 months for the latest releases of each major version

I would suggest you update your Rocket.Chat version to the latest stable version, which you can find in [Releases Page](https://github.com/RocketChat/Rocket.Chat/releases) and, check it [HERE](https://docs.rocket.chat/resources/get-support/enterprise-support#rocket.chat-versions) the table with versions and End of Life dates. 

---

**Questions? Help needed? Feature Requests?** 

- Join our [Open Server](https://open.rocket.chat/) in the #support channel and feel free to raise a question
- Join our [Community Forum](https://forums.rocket.chat/) and search/create a post there
